### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mithodin**
Added documentation for SMTP auth.

### CODE REVIEW

**Comment by moy** [comment]
@mithodin: Thanks for the update. But would you please add a `Signed-off-by` line to the commit, indicating your acceptance of the license [in the manner of the Git project](https://github.com/git/git/blob/9532ead9875b7b92ff2007f9de61af5874e8839a/Documentation/SubmittingPatches#L234)? Feel free to force-push to your branch with the fixed commit. Thanks!

File: git-multimail/README [hunk 1]
```diff
@@ -263,6 +263,29 @@ multimailhook.mailer
...
+
+           The port on the SMTP server to connect to. Usage of TLS/SSL
+           is detected automatically.
+           	Port 25: No Encryption (not recommended)
+           	Port 465: Use SSL
+           	Port 587: Use StartTLS
+
+       multimailhook.smtpAuth
+
+           Wether or not to use authentication with the server.
```
Comment (mhagger) [inline git-multimail/README]: Typo: s/Wether/Whether/

**Comment by mithodin** [comment]
It's signed off now (and the typo is fixed)

**Comment by moy** [comment]
I pushed a branch smtp-doc containing both commits squashed. But this would only make sense when #63 is merged, and for now it lacks a signed-off-by.

**Comment by moy** [comment]
Actually, I'll merge #85 soon, hence I'm closing this one.

**Pull request closed without merging**